### PR TITLE
Add Shopify connector

### PR DESCRIPTION
**Pull request by RolandHewage**
# Description

As Shopify API doesn't has an existing official OpenAPI specification, we wrote the OpenAPI specification and generate Shopify Admin API connector using the hand written Open API specification.

https://shopify.dev/api/admin

Market research document
https://docs.google.com/spreadsheets/d/1wh3Qo5SFKhZofnxqiN-ZAS-4TREnmujO7sf4pCre8-Y/edit?usp=sharing

Resolves https://github.com/ballerina-platform/ballerina-extended-library/issues/96
Fixes # (https://github.com/ballerina-platform/ballerina-extended-library/issues/82)

One line release note: 
- Generate Shortcut connector using Open API specification. 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

**Test Configuration**:
* Ballerina Version: Ballerina Swan Lake Beta3
* Operating System: Ubuntu 20.04
* Java SDK: JDK 11

# Checklist:

### Changes to OpenAPI definition

- [x] Modified `info` section of the OpenAPI definition file - [Example](https://github.com/ballerina-platform/ballerina-extended-library/discussions/74)

    - `description` - what the connector is about 
    - `x-ballerina-init-description` OpenAPI extension - connector initialization details 
    - `x-ballerina-display` OpenAPI extension - connector name and path to the connector icon 

- [x] Added proper description each API Key, if API Key authentication is defined in OpenAPI definition file 

### Changes to connector module

- [x] Added license header at the top of each .bal file. 
- [x] Added Package.md as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/77)
- [x] Added Module.md for the module of the  connector as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/78)
- [x] Created a directory called `resources` at the root of the connector and copied the connector icon there. 
      Name of the icon file need to be `<connector-name>.svg`
- [x] Added Ballerina.toml file with following information. For keywords refer to [guide](https://github.com/ballerina-platform/ballerina-extended-library/discussions/72)

### Verifying connector module

- [x] Compiled the connector successfully with the targeted ballerina version. 
- [x] Conducted smoke tests on the connector (Recommended when OpenAPI definition is obtain from an unofficial source)

### Security checks
 - [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: openapi/shopify.admin/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+org = "ballerinax"
+name = "shopify.admin"
+version = "0.1.0-SNAPSHOT"
```
Comment (LakshanSS) [inline openapi/shopify.admin/Ballerina.toml:4]: Change version to `1.0.0-SNAPSHOT`

File: openapi/shopify.admin/Package.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+This package provides the capability to access Shopify Admin API.
+
+### Compatibility
+|                               | Version                         |
+|-------------------------------|---------------------------------|
+| Ballerina Language            | Ballerina Swan Lake Beta3       | 
+| Shopify Admin API             | 2021-10                         |
+
+## Report issues
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina connector repository](https://github.com/ballerina-platform/ballerinax-openapi-connectors)
```
Comment (LakshanSS) [inline openapi/shopify.admin/Package.md:14]: This has to point to extended-library repo

File: openapi/shopify.admin/Ballerina.toml (hunk 1 above)
Comment (RolandHewage) [reply openapi/shopify.admin/Ballerina.toml]: Resolved. Thank you.

File: openapi/shopify.admin/Package.md (hunk 2 above)
Comment (RolandHewage) [reply openapi/shopify.admin/Package.md]: Resolved. Thank you.

**Review by aneeshafedo — APPROVED** [review]

**Review by LakshanSS — APPROVED** [review]

**Comment by SkNuwanTissera** [comment]
Icon for this connector is available now [1].
You can download & locate the SVG file inside resources
[1] https://drive.google.com/drive/u/1/folders/1W3qKG3V5OWo8XfJFuqw9yod9wG4qd2LD

**Pull request merged**